docker-compose.yml

```diff
@@ -11,7 +11,7 @@ services:
       MB_PREMIUM_EMBEDDING_TOKEN: "${PREMIUM_EMBEDDING_TOKEN}"
       MB_RUN_MODE: "${MB_RUN_MODE}"
       METASTORE_DEV_SERVER_URL: "${METASTORE_DEV_SERVER_URL}"
-      MB_JWT_IDENTITY_PROVIDER_URI: "http://localhost:${AUTH_PROVIDER_PORT}/sso/metabase"
+      MB_JWT_IDENTITY_PROVIDER_URI: "http://unused.com/sso/metabase"
```

Comment:
I suspect this env variable has been removed for a while. So I changed it to reflect that this value isn't really used as we set `fetchRequestToken`.
https://github.com/metabase/metabase-nextjs-sdk-embedding-sample/blob/16af3fa8e7d70a4a05e2c414fa156effdc8548cb/next-sample-app-router/src/app/app-provider.tsx#L16-L20

We still require it to be set, though.

You can see [on the CI](https://github.com/metabase/metabase-nextjs-sdk-embedding-sample/actions/runs/20489143861/job/58877663796#step:3:16) that this variable hasn't been set.

> time="2025-12-24T15:25:16Z" level=warning msg="The \"AUTH_PROVIDER_PORT\" variable is not set. Defaulting to a blank string."